.buildkite/core.rayci.yml

```diff
@@ -315,3 +315,14 @@ steps:
       - forge
       - raycpubase
       - corebuild
+
+  - label: ":ray: core: multi gpu tests"
+    tags:
+      - python
+      - gpu
+    instance_type: gpu-large
+    commands:
+      - bazel run //ci/ray_ci:test_in_docker -- //python/ray/tests/... //python/ray/dag/... core
+        --gpus 2
```

Comment:
perhaps `--parallelism-per-worker 2 --gpus 4`; this machine has 4 gpus, we will run two tests in parallel at most and give each test 2 gpu max

Reply:
Got it, thanks!